modules/openapi-generator/src/main/resources/php-nextgen/ObjectSerializer.mustache

```diff
@@ -529,22 +529,63 @@ class ObjectSerializer
     }
 
     /**
-     * Native `http_build_query` wrapper.
-     * @see https://www.php.net/manual/en/function.http-build-query
-     *
-     * @param array|object $data           May be an array or object containing properties.
-     * @param string       $numeric_prefix If numeric indices are used in the base array and this parameter is provided, it will be prepended to the numeric index for elements in the base array only.
-     * @param string|null  $arg_separator  arg_separator.output is used to separate arguments but may be overridden by specifying this parameter.
-     * @param int          $encoding_type  Encoding type. By default, PHP_QUERY_RFC1738.
-     *
-     * @return string
-     */
-    public static function buildQuery(
-        array|object $data,
-        string       $numeric_prefix = '',
-        ?string      $arg_separator = null,
-        int          $encoding_type = \PHP_QUERY_RFC3986
-    ): string {
-        return \GuzzleHttp\Psr7\Query::build($data, $encoding_type);
+    * Build a query string from an array of key value pairs.
+    *
+    * This function can use the return value of `parse()` to build a query
+    * string. This function does not modify the provided keys when an array is
+    * encountered (like `http_build_query()` would).
+    *
```

Comment:
can you please add a 1-liner indicating this function is based on https://github.com/guzzle/psr7/blob/540ec79fb2f2e96591febb0de464bc16ac2ea341/src/Query.php#L73 ?